src/components/DetailCard/DetailCard.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import styles from './DetailCard.css';
+
+export default function Detail({ selected, select, handleClick }) {
+  const {
+    id,
+    image,
+    name,
+    gender,
+    firstEpisode,
+    hairColor,
+    occupation,
+    season,
+    episode,
+    airDate,
+    totalViewers,
+    productionCode,
```

Comment:
It might be easier just to spread selected in when you pass it as props to this component and you can destructure it on line 4